skipper/cli.py

```diff
@@ -104,12 +105,21 @@ def push(ctx, namespace, image):
         utils.logger.error('Failed to tag image: %(tag)s as fqdn', dict(tag=image_name, fqdn=fqdn_image))
         sys.exit(ret)
 
-    utils.logger.debug("Pushing to registry %(registry)s", dict(registry=ctx.obj['registry']))
-    command = ['docker', 'push', fqdn_image]
-    ret = runner.run(command)
-    if ret != 0:
-        utils.logger.error('Failed to push image: %(tag)s', dict(tag=fqdn_image))
-        sys.exit(ret)
+    repo_name = utils.generate_fqdn_image(None, namespace, image, tag=None)
+
+    if not force:
```

Comment:
same comment as in the package-builder. If force, lets do the verification and print the warning anyhow